dist/GoldRush.js

```diff
@@ -0,0 +1,138 @@
+class GoldRush extends Matrix {
+    constructor(rowNum, columnNum) {
+        super(rowNum, columnNum)
+        this.players = [{ num: 1, score: 0 }, { num: 2, score: 0 }]
+        this.rowNum = parseInt(rowNum)
+        this.columnNum = parseInt(rowNum)
+        this.numC = Math.round(0.33 * (parseInt(rowNum) * parseInt(rowNum)))
+        this.numWall = Math.round(0.2 * (parseInt(rowNum) * parseInt(rowNum)))
+    }
+    loadBoard() {
+        this.matrix[0][0] = this.players[1].num
+        this.matrix[this.rowNum - 1][this.columnNum - 1] = this.players[0].num
+
+        let boardLoaded = false
+        let wallsRemain = this.numWall
+        let coinsRemain //Checking if number of coins is odd, to ensure a winner:
+        this.numC % 2 == 0 ? coinsRemain = this.numC-1 : coinsRemain = this.numC 
```

Comment:
`coinsRemain =  this.numC % 2 == 0 ? this.numC-1 : this.numC`

same thing but shorter